src/components/Footer.vue

```diff
@@ -31,7 +31,7 @@
         v-if="!phoneLayout"
       />
       <div class="icon-label">
-        <v-icon>mdi-sprout</v-icon>
+        <v-icon>mdi-format-list-text</v-icon>
```

Comment:
reject all changes in this file

Reply:
What are the issues again? There's much more context now.